openpype/lib/applications.py

```diff
@@ -1490,6 +1490,11 @@ def _prepare_last_workfile(data, workdir):
     import avalon.api
 
     log = data["log"]
+
+    if not data.get("start_last_workfile", True):
```

Comment:
You can't skip collection of important data for prelaunch hooks...

Reply:
Modified, is this better?